### ISSUE TITLE
Questions about loss fonction

### ISSUE THREAD

**Issue by txkthu** [imanolperez/market_simulator#2]
Hello I am a student in applied mathematics and I'm interested in one of your works "A DATA-DRIVEN MARKET SIMULATOR
FOR SMALL DATA ENVIRONMENTS". 

Could you please give more details on the latent loss function in the CVAE.py file, what is the intuition? Thanks a lot!
The corresponding code runs as follows.
`decoded_loss = tf.reduce_sum(tf.squared_difference(unreshaped, Y_flat), 1)`
`latent_loss = -0.5 * tf.reduce_sum(1. + sd - tf.square(mn) - tf.exp(sd), 1)`
